modules/imgcodecs/src/loadsave.cpp

```diff
@@ -518,6 +518,29 @@ bool imreadmulti(const String& filename, std::vector<Mat>& mats, int flags)
     return imreadmulti_(filename, flags, mats);
 }
 
+static void
+convertToCV8U(const Mat& src, const Mat& dst, int rtype)
+{
+    switch (rtype)
```

Comment:
why not simply write `src.depth()` here instead of `rtype`? We can eliminate `rtype` parameter